Content.Server/GameObjects/Components/Damage/DestructibleComponent.cs

```diff
@@ -19,45 +19,42 @@ public class DestructibleComponent : Component, IOnDamageBehavior
         /// <inheritdoc />
         public override string Name => "Destructible";
 
-        /// <inheritdoc />
-        public override uint? NetID => ContentNetIDs.DESTRUCTIBLE;
-
         /// <summary>
         /// Damage threshold calculated from the values
         /// given in the prototype declaration.
         /// </summary>
         [ViewVariables]
         public DamageThreshold Threshold { get; private set; }
```

Comment:
Should support multiple of these too.
Would just need to make `DamageThreshold` serializable.